pkg/_internal/metadata/finalizers.go

```diff
@@ -1,53 +0,0 @@
-package metadata
```

Comment:
This looks to be removed in favour of `controllerutil`?  I wonder is there something similar we could use to replace annotations and labels at some point too.

Reply:
Yes i removed this since there were helper functions in controller runtime that did the exact same thing. Not sure about annotations though tbh.